src/app/core/shared/shared.module.ts

```diff
@@ -4,23 +4,26 @@ import { CommonModule } from '@angular/common';
 
```

Comment:
It's good approach to separate core module from shared module. Best practice is to have app module (main module of your application), core module (which includes app component only related components and parts), shared module (includes parts which can be used in across application: common components, directives and pipes) and feature modules ( view specific components and logic parts). Services are exception. You need to provide services in main module, using providers array.